services/app-api/forms/mcpar.json

```diff
@@ -202,7 +202,7 @@
                     },
                     {
                       "type": "html",
-                      "content": "See Glossary in Excel Workbook for the definition of BSS entities."
+                      "content": " See Glossary in Excel Workbook for the definition of BSS entities."
```

Comment:
Change this to `". See Glossary in Excel Workbook for the definition of BSS entities."`?

Right now it renders as:

<img width="669" alt="Screenshot 2024-12-06 at 1 29 00 PM" src="https://github.com/user-attachments/assets/04891278-2834-438f-a545-9cb4d64c4b1f">